packages/components/src/blob/animation.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useCallback, useEffect, useRef, memo } from "react";
+import { canvasPath as animCanvasPath } from "blobs/v2/animate";
+import { BlobProps } from "./base";
+import { colors } from "@variant/profile";
+import { useIsMountedRef } from "../utils/useMounted";
+import { useReducedMotion } from "framer-motion";
+import { BlobOptions } from "blobs/v2";
+
+type AnimatedBlobProps = BlobProps & {
+  animationSpeed?: number;
+};
+
+export const AnimatedBlob: React.FC<AnimatedBlobProps> = memo(
+  ({
+    width,
+    height,
+    seed = Math.random(),
+    extraPoints = 4,
+    randomness = 9,
+    color = colors.colorPairs.primary.default.bg.toString(),
```

Comment:
Isn't this already a string, i.e. no need for `toString()`?

Reply:
Jepp, mente jeg hadde fikset det til å ta imot `colorPairs.primary.default` type.